src/shared/browser-compatibility-utils.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Normalize a keyboard event key name.
+ *
+ * IE11 uses alternate key names for its keyboard events.
+ * If any abnormal keys are used (via IE11), this method
+ * returns the normalized name so our UI components
+ * don't require a special case.
+ *
+ * @param {string} key - The keyboard event `key` name
+ * @return {string} - Normalized `key` name
+ */
+export function getNormalKey(key) {
```

Comment:
Can I suggest `normalizeKeyName` as a name for this? I think it may not be obvious to a reader what a "normal" key is (as opposed to _normalizing_ the key name).